pandas/_libs/tslib.pyx

```diff
@@ -275,8 +273,6 @@ def array_with_unit_to_datetime(
         )
         return result, tz
 
-    mult, _ = precision_from_unit(unit)
```

Comment:
😄 is this unused in this function? I'd have hoped cython-lint would've caught this, I'll see what I can do there

Reply:
correct